### ISSUE TITLE
Can't Download due to Colons

### ISSUE THREAD

**Issue by AppleCanon** [ErnestoArtigas/khInsiderBot#1]
At least im assuming its due to that. Im trying to download (https://downloads.khinsider.com/game-soundtracks/album/phoenix-wright-ace-attorney-arranged-by-jattello-2015) and the name is "Phoenix Wright: Ace Attorney (Arranged by jattello)". Its trying to create a folder for it but keeps saying its invalid - **[WinError 267] The directory name is invalid: 'C:\\Users\\libre\\Downloads\\khInsiderBot-main\\Phoenix Wright: Ace Attorney (Arranged by jattello) (2015)'**. Folders dont let you use : in them so im assuming its because of that. Anyway to change how the downloader does names or ignore certain symbols?

**Comment by ErnestoArtigas** [comment]
That's very strange, the link you've sent uses a - and not a : 
I've downloaded the link and it creates the folder.

```sh
python main.py -f mp3 -l https://downloads.khinsider.com/game-soundtracks/album/phoenix-wright-ace-attorney-arranged-by-jattello-2015`
```

How did you use the program ?

**Comment by AppleCanon** [comment]
> That's very strange, the link you've sent uses a - and not a : I've downloaded the link and it creates the folder.
> 
> ```shell
> python main.py -f mp3 -l https://downloads.khinsider.com/game-soundtracks/album/phoenix-wright-ace-attorney-arranged-by-jattello-2015`
> ```
> 
> How did you use the program ?

Ok, so I made on edit on the download.khinsider page. It WAS "Phoenix Wright: Ace Attorney" but I made it a -, so it appears a mod recently approved the edit. I redid the command it it works now, so im assuming it WAS due to the colon. To answer your question, I use the program but going to the folder (in my case, khInsiderBot-main in my Downloads) and pressing "Open in Terminal" when in it. I then put the following code from above in the command line. The program reads it and then states its loaded but then fails. Weirdly after playing around with it, it started giving me double slashes. For example: [WinError 267] The directory name is invalid: 'C:\\Users\\libre\\Downloads\\khInsiderBot-main\\Phoenix Wright: Ace Attorney (Arranged by jattello) (2015)'. But as I said it appears to be working, just need some work around for the colon. Here's another link that has a colon in the name: (https://downloads.khinsider.com/game-soundtracks/album/the-spongebob-squarepants-movie-video-game-soundtrack)
